pkg/azurefileplugin/Dockerfile

```diff
@@ -18,17 +18,17 @@ ARG ARCH=amd64
 ARG binary=./_output/${ARCH}/azurefileplugin
 COPY ${binary} /azurefileplugin
 
-RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates cifs-utils util-linux e2fsprogs mount udev xfsprogs nfs-common netbase curl
+RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates cifs-utils util-linux e2fsprogs mount udev xfsprogs nfs-common netbase wget
 
 # install azcopy
-ARG azcopyURL=https://aka.ms/downloadazcopy-v10-linux
+ARG azcopyURL=https://azcopyvnext.azureedge.net/releases/release-10.21.2-20231106/azcopy_linux_amd64_10.21.2.tar.gz
 RUN if [ "$ARCH" == "arm64" ] ; then \
-  azcopyURL=${azcopyURL}"-arm64"; fi
-RUN curl -sL ${azcopyURL} | tar -xz && \
+  azcopyURL=https://azcopyvnext.azureedge.net/releases/release-10.21.2-20231106/azcopy_linux_arm64_10.21.2.tar.gz; fi
+RUN wget -O azcopy.tar.gz ${azcopyURL} && \
+  tar xvzf azcopy.tar.gz -C . && rm azcopy.tar.gz && \
   mv ./azcopy_linux_$ARCH_*/azcopy /usr/local/bin/azcopy && \
   rm -rf ./azcopy_linux_$ARCH_*
 RUN chmod +x /usr/local/bin/azcopy
-RUN apt remove curl -y
```

Comment:
if you are using wget, you need to remove wget finally

Reply:
done